moped-editor/src/queries/tableLookups.js

```diff
@@ -23,14 +23,20 @@ export const TABLE_LOOKUPS_QUERY = gql`
         phase_name
       }
     }
-    moped_components {
+    moped_components(
+      order_by: [{ component_name: asc }, { component_subtype: asc }]
+    ) {
```

Comment:
fixes the random sorting currently in prod.

Reply:
Thank you! Consistency in the app interface from visit to visit is quality polish.

Reply:
✨